admin_ui/components/Nav.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+/**
+ * LHS navigation.
+ * TODO: use #top for link hrefs here to avoid eslint's "anchor-is-valid" perstering us,
+ *  change this when we have proper routes.
+ */
+const Nav = () => {
```

Comment:
I would opt-in for a more specific name, e.g. `<AsideNav />`

Reply:
Nice! done.